src/Init/Data/Iterators/Lemmas/Consumers/Collect.lean

```diff
@@ -144,4 +144,13 @@ theorem Iter.isPlausibleIndirectOutput_of_mem_toListRev
   apply isPlausibleIndirectOutput_of_mem_toList
   simpa [toListRev_eq] using h
 
+theorem Iter.isPlausibleIndirectOutput_of_mem_toArray
+    [Iterator α Id β] [Finite α Id] [IteratorCollect α Id Id] [LawfulIteratorCollect α Id Id]
+    {it : Iter (α := α) β} {b : β} :
+    b ∈ it.toArray → it.IsPlausibleIndirectOutput b := by
+  intro h
+  apply isPlausibleIndirectOutput_of_mem_toList
+  rw [←Array.mem_toList_iff] at h
```

Comment:
Just a small syntactical remark: In the standard library, we usually put a space after  `←` in `simp` calls.

```suggestion
  rw [← Array.mem_toList_iff] at h
```